app/src/instantsearch.js

```diff
@@ -42,6 +42,7 @@ class InstantSearch {
         }
       },
       searchParameters: {
+        analytics,
```

Comment:
I might be reading that wrong, but I believe you're not getting `analytics` in the function parameters (but are doing so in the `render` function).
You'll need to remove it from `render` and put it in this `constructor`.